src/main/java/model/Player.java

```diff
@@ -245,6 +246,27 @@ public void die() {
     }
 
     /**
+     * This method will remove a bubble from the ArrayList of Bubbles shot by the Player.
+     */
+    public void checkBubbles() {
+
+        Iterator<Bubble> i = bubbles.iterator();
+
+        while (i.hasNext()) {
+
+            Bubble bubble = i.next();
+            if (bubble.checkPop() && !bubble.getIsPrisonBubble()) {
+
+                i.remove();
+                levelController.getScreenController().removeSprite(bubble);
+
+                Logger.log(String.format("Bubble is popped"));
```

Comment:
This String.format(...) is not needed because no String is really formatted. ```Logger.log("Bubble is popped");``` would suffice.